core/CronArchive.php

```diff
@@ -228,8 +228,6 @@ public function __construct($processNewSegmentsFrom = null, LoggerInterface $log
         $this->logger = $logger ?: StaticContainer::get('Psr\Log\LoggerInterface');
         $this->formatter = new Formatter();
 
-        $this->processNewSegmentsFrom = $processNewSegmentsFrom ?: StaticContainer::get('ini.General.process_new_segments_from');
```

Comment:
Seems that property is no longer in use. Guess it could be removed as property and maybe also as constructor parameter